moma_controllers/moma_joint_velocity_controller/src/controller.cpp

```diff
@@ -55,9 +55,9 @@ bool JointVelocityController::init(hardware_interface::RobotHW* hw, ros::NodeHan
     return false;
   }
 
-  if (!controller_nh.getParam("/robot_description", robot_description_) ||
-      robot_description_.empty()) {
-    ROS_ERROR_STREAM("Could not find param /robot_description or invalid param");
+  if (!controller_nh.getParam("/arm_description", arm_description_) ||
```

Comment:
Well, using `arm_description` does not mean you cannot have a `robot_description` on the param server. Then it is more a matter of naming convention

Reply:
Yes, but the default ROS controllers use `robot_description`. I'll need to check if this will still work then (as you needed this change in the moma_joint_velocity_controller, I guess that some of the default ROS controllers may also need changes, whereas if we had `robot_description` only the arm, the default controllers would work as is).
I'll test the UR later on and then we can see how to proceed.